model.py

```diff
@@ -0,0 +1,69 @@
+import glob
+from pathlib import Path
+from typing import Dict, List
+from dataclasses import dataclass
+import csv
+
+@dataclass
+class Criterion:
+    name: str
+    description: str
+    scores: List[str]
+
+@dataclass
+class Applicant:
+    name: str
+    cv: str #path to cv
+    scores: Dict[Criterion,str]
+
+@dataclass
+class Role:
+    job_title: str
+    job_id: str
+    criteria: List[Criterion]
+
+
+@dataclass
+class Shortlist:
+    role: Role
+    applicants: List[Applicant]
+
+
+#functions
+
+def load_shortlist(path):
```

Comment:
This file looks good. It's a good idea to write a short comment about what a function does, especially what kind of argument is needed to call the function, and what does it return.

docstrings look like this:

```
def my_function(arg):
    """this function takes an integer, adds one and returns the result"""
    x = arg + 1
    return x
```